.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+
+Thank you for collating and sharing your knowledge!
+
+PR Title Format:
+	 [date]: [topic]
+e.g. 2022-05-16: database introduction
+
+-->
+
+### Summary of talk
+
+<!--
+The summary of talk, feel free to add anything more.
+-->
+Date:
+
+Topic:
+
+Keywords:
+
+### Format check List
+
+- [ ] Conforming to English specifications especially in `README.md`.
+- Capitalize the first letter.
+- Use English punctuation with appropriate space.
+- ...
+- [ ] Remember to add links to reference papers.
+- [ ] Make sure the link is accessible.
+- Use `-` to connect filename rather than ` `, the latter will casue dead link.
```

Comment:
rather than `,`

Reply:
I mean space ` ` here.
```txt
[abc file](a bc.pdf) # dead link
[abc file](a-bc.pdf) # live link
```

Reply:
Change to in new commit:
 Use dash `-` to connect filename rather than space ` `, the latter will casue dead link.